hardening/host-os/ansible/main.fmf

```diff
@@ -4,6 +4,8 @@ result: custom
 environment+:
     PYTHONPATH: ../../..
 duration: 1h
+require+:
+  - openscap-scanner
```

Comment:
I mean - having `host-os/main.fmf` crossed my mind later after creating this PR and it's definitely something we could do. It hides the require a little bit (people don't always look in parent dirs for `.fmf`), but it also deduplicates code.

OTOH it makes the tests a bit harder to copy/move around or use as templates for new tests.

Honestly, it's up to you, I can move the require if needed.

Reply:
Let's leave it as it is. At least for now. If there would be more tests where we could put requirements (or something else), I'd consider this. But now, as it would be the only case, it would rather complicate things than make them simpler.